install_config/persistent_storage/persistent_storage_manila.adoc

```diff
@@ -0,0 +1,270 @@
+[[installation-setup]]
+== Installation and Setup
+
+The featuere is provided by an external provisioner that needs to be installed and
+configured in the {product-title} cluster.
+
+[[start-controller-and-provisioner]]
+=== Starting the External Provisioner
+
+The external provisioner service is distributed as a container image and can be run in the
+{product-title} cluster as usual. There is also RPM versions for provisioner.
+
+To allow the containers managing the API objects, the necessary role-based access control
+(RBAC) rules need to be configured by the administrator:
+
+. Create a `ServiceAccount`:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: manila-provisioner-runner
+----
+
+. Create a `ClusterRole`:
++
+[source,yaml]
+----
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: manila-provisioner-role
+rules:
+  - apiGroups: [""]
+    resources: ["persistentvolumes"]
+    verbs: ["get", "list", "watch", "create", "delete"]
+  - apiGroups: [""]
+    resources: ["persistentvolumeclaims"]
+    verbs: ["get", "list", "watch", "update"]
+  - apiGroups: ["storage.k8s.io"]
+    resources: ["storageclasses"]
+    verbs: ["get", "list", "watch"]
+  - apiGroups: [""]
+    resources: ["events"]
+    verbs: ["list", "watch", "create", "update", "patch"]
+----
+
+. Bind the rules via `ClusterRoleBinding`:
++
+[source,yaml]
+----
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: manila-provisioner
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: manila-provisioner-role
+subjects:
+- kind: ServiceAccount
+  name: manila-provisioner-runner
+  namespace: default
+----
+
+. Create a new `StorageClass`:
++
+[source,yaml]
+----
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: "manila-share"
+provisioner: "externalstorage.k8s.io/manila"
+parameters:
+  type: "default"
+  zones: "nova"
+----
+
+The `type` parameter specifes the
+link:https://docs.openstack.org/manila/latest/admin/shared-file-systems-share-types.html[Manila share type]
+the provisioner will create for the volume. The `zones` parameter is a set of
+Manila availability zones the volume might be created in.
+
+The provisioner must be able to connect, authenticate, and authorize to the
+Manila service. The connection is configured using environment variables. There
+are several ways to do so. The following environment variable combinations might
+be used:
+
+[source]
+----
+OS_USERNAME
+OS_PASSWORD
+OS_AUTH_URL
+OS_DOMAIN_NAME
+OS_TENANT_NAME
+----
+
+[source]
+----
+OS_USERID
+OS_PASSWORD
+OS_AUTH_URL
+OS_TENANT_ID
+----
+
+[source]
+----
+OS_USERNAME
+OS_PASSWORD
+OS_AUTH_URL
+OS_DOMAIN_ID
+OS_TENANT_NAME
+----
+
+[source]
+----
+OS_USERNAME
+OS_PASSWORD
+OS_AUTH_URL
+OS_DOMAIN_ID
+OS_TENANT_ID
+----
+
+It is recommended to use `Secret` to pass the variables to the provisioner.
+The example of the `Secret` for the first variables combination:
```

Comment:
s/The example of the `Secret` for the first variables combination:/. Example of the `Secret` for the first variables combination

Reply:
The `Secret` is used by an example below. So it's not only the `Secret`. Caption would be confusing I think.